src/creators/createReducer/test.js

```diff
@@ -0,0 +1,56 @@
+import Immutable from 'seamless-immutable';
+
+import createReducer from '.';
+
+const initialState = {
+  count: 0
+};
+
+const setUp = {
+  state: null
+};
+
+const actions = { DECREMENT: 'decrement', INCREMENT: 'increment', DUMMY: 'dummy' };
+
+const dummyAction = { type: actions.DUMMY };
+const decrementAction = { type: actions.DECREMENT, target: 'count' };
+const incrementAction = { type: actions.INCREMENT, target: 'count' };
+
+beforeEach(() => {
+  setUp.state = Immutable(initialState);
+});
+
+describe('createReducer', () => {
+  it('Throws if no reducer description is passed', () => {
+    expect(() => createReducer({}, null)).toThrowError(Error, 'Expected a reducer description as an object.');
+  });
+  it('Throws if a non object reducer description is passed', () => {
+    expect(() => createReducer({}, [])).toThrowError(Error, 'Expected a reducer description as an object.');
+  });
+  it('Initializes state correctly', () => {
+    const reducerDescription = {};
+    const reducer = createReducer(setUp.state, reducerDescription);
+    expect(reducer(setUp.state, dummyAction).count).toBe(setUp.state.count);
```

Comment:
Mmm I'm not sure if this is the way to test that the state is initialized correctly.